framework/test/jest/integration/specs/modules/chain/components/storage/entities/chain_meta.spec.js

```diff
@@ -0,0 +1,248 @@
+/* eslint-disable mocha/no-pending-tests */
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+'use strict';
+
+const {
+	StorageSandbox,
+} = require('../../../../../../../../mocha/common/storage_sandbox');
+const seeder = require('../../../../../../../../mocha/common/storage_seed');
+const {
+	entities: { BaseEntity },
+} = require('../../../../../../../../../src/components/storage');
+const {
+	ChainMeta,
+} = require('../../../../../../../../../src/modules/chain/components/storage/entities');
+
+const { config } = require('../../shared');
+
+const getAllMeta = async adapter =>
+	adapter.execute('select * from chain_meta;');
+
+describe('ChainMeta', () => {
+	const storage = new StorageSandbox(
+		config.components.storage,
+		'lisk_test_chain_module_storage_chain_meta'
+	);
+	let adapter;
+	let ChainMetaEntity;
```

Comment:
Prefer consistent ordering of `const` and `let`

Reply:
Didn't get it.

Reply:
Sorry 😉 Instead of

```
const x = '';
let y = '';
const z = 3;
```

prefer below
```
const x = '';
const z = 3;
let y = '';
```